docs/rules/vars-on-top.md

```diff
@@ -7,9 +7,11 @@ This rule forces the programmer to represent that behaviour by manually moving t
 ## Rule Details
 
 This rule aims to keep all variable declarations in the leading series of statements.
-Allowing multiple helps promote maintainability and reduces syntax.
+Allowing multiple declarations helps promote maintainability and is thus allowed.
 
-No variable declarations in a block.
+### Examples
+
+#### No variable declarations in a block.
```

Comment:
Unnecessary headers here. Let's not have one-off headers... these don't appear in any other rule doc.